backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -224,8 +226,19 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
   }
 
   /** The instantiated command. */
-  lazy val instantiatedCommand: String = Command.instantiate(
-    jobDescriptor, backendEngineFunctions, commandLinePreProcessor, commandLineValueMapper).get
+  lazy val instantiatedCommand: String =  {
+
+    val todoMoveMeMinimums =
+      MinimumRuntimeSettings(
+        cores = 1,
+        ram = MemorySize.apply(4, MemoryUnit.GiB),
+        outputPathSize = Long.MaxValue,
+        tempPathSize = Long.MaxValue
+      )
```

Comment:
Isn't this `MinimumRuntimeSettings()` ?

Reply:
whoops, I meant to remove this and take in the argument :)